src/coreclr/vm/syncblk.cpp

```diff
@@ -532,6 +534,10 @@ SyncBlockCache::~SyncBlockCache()
     }
 }
 
+#ifdef TARGET_LOONGARCH64
```

Comment:
Is it possible to use the LOONGARCH64 version everywhere without ifdef?

Reply:
> Is it possible to use the LOONGARCH64 version everywhere without ifdef?

OK, here is just for optimize.
If need, I can rewrite it.